examples/foam/index.html

```diff
@@ -2,11 +2,10 @@
 <head>
   <meta charset="utf-8">
   <title>FOAM • TodoMVC</title>
-  <link rel="stylesheet" href="bower_components/todomvc-common/base.css">
-  <link rel="stylesheet" href="bower_components/todomvc-app-css/index.css">
-  <script src="bower_components/foam/build/foam.js"></script>
-  <script src="bower_components/todomvc-common/base.js"></script>
-  <script src="Todo.js"></script>
+  <link rel="stylesheet" href="node_modules/todomvc-common/base.css">
+  <link rel="stylesheet" href="node_modules/todomvc-app-css/index.css">
+  <script language="javascript" src="node_modules/todomvc-common/base.js"></script>
```

Comment:
Looks like it is still in the spec
http://www.w3.org/TR/html4/interact/scripts.html#h-18.2.2

But it looks like since we specify the content type in the header we should not need this.

> Otherwise, if any HTTP headers specify the "Content-Script-Type", the last one in the character stream determines the default scripting language.